shrinker-test/src/main/java/com/example/NoSerializedNameMain.java

```diff
@@ -19,25 +19,25 @@ static class TestClassNotAbstract {
     public String s;
   }
 
-  static class TestClassWithoutDefaultConstructor {
+  static class TestClassConstructorHasArgs {
     public String s;
 
     // Specify explicit constructor with args to remove implicit no-args default constructor
-    public TestClassWithoutDefaultConstructor(String s) {
+    public TestClassConstructorHasArgs(String s) {
       this.s = s;
     }
   }
 
   /**
-   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_DefaultConstructor()}.
+   * Main entrypoint, called by {@code ShrinkingIT.testNoSerializedName_ConstructorHasArgs()}.
```

Comment:
Or actually, maybe this specific instance should not have been renamed because `TestClass` does actually have an implicit default constructor.

Reply:
I think I like NoArgsConstructor better because the code doesn't really care if it's default or not.

Reply:
Adjusted the test method/class names for consistency.